src/rock-gazebo.in

```diff
@@ -1,4 +1,14 @@
-#! /bin/sh
+#! /usr/bin/ruby
+require 'rock/bundles'
+require 'optparse'
 
-gazebo --verbose -s @CMAKE_INSTALL_PREFIX@/lib/librock_gazebo.so "$@"
+options = OptionParser.new
+args = options.parse(ARGV)
+world_file = args.shift
 
+Bundles.load
+
+ENV['GAZEBO_MODEL_PATH']=(ENV['GAZEBO_MODEL_PATH']||"")+
```

Comment:
why the order is important ? 
it is not unlikely to have our bundle world file inside another gazebo path ?